CRP.Mvc/Views/Item/DetailsNew.cshtml

```diff
@@ -38,7 +39,7 @@
 @section AdditionalScripts
 {
     <script src="https://maps-api-ssl.google.com/maps/api/js?v=3&sensor=false&key=@CloudConfigurationManager.GetSetting("MapsApi")" type="text/javascript"></script>
-    
+    <link href="@Url.Content("~/Content/jquery.gPositions.css")" rel="Stylesheet" type="text/css" />

```

Comment:
@jSylvestre you should put this up top before the content in additional styles instead